atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -439,6 +439,13 @@ def get_operator_manifests(workflow, fallback=NO_FALLBACK):
     return get_value(workflow, 'operator_manifests', fallback)
 
 
+def get_image_size_limit(workflow):
+    config = get_value(workflow, 'image_size_limit', {})
+    return {
+        'regular_image': config.get('regular_image', 0),
```

Comment:
Are we use regular to mean binary here?

If so, we should rename it to binary_image, since this is the name we have been using everywhere else to mean non-source-images.

Reply:
Done.